pkg/model/stack.go

```diff
@@ -366,6 +369,29 @@ func validateStackName(name string) error {
 	return nil
 }
 
+func validateDependsOn(s *Stack) error {
+	for svcName, svc := range s.Services {
+		for dependentSvc, condition := range svc.DependsOn {
+			if svcName == dependentSvc {
+				return fmt.Errorf(" Service '%s' depends can not depend of itself.", svcName)
+			}
+			if _, ok := s.Services[dependentSvc]; !ok {
+				return fmt.Errorf(" Service '%s' depends on service '%s' which is undefined.", svcName, dependentSvc)
+			}
+			if condition.Condition == DependsOnServiceCompleted && !s.Services[dependentSvc].IsJob() {
+				return fmt.Errorf(" Service '%s' is not a job. Please change the reset policy so that it is not always in service '%s' ", dependentSvc, dependentSvc)
```

Comment:
I don't understand this sentence:
```
Please change the reset policy so that it is not always in service
```

Reply:
It means that they have set a `DependsOnServiceCompleted` with a service with a restart policy set to `always`. We only support that kind of condition with jobs where the restart policy can not be `always`.
It was the error that we had but it's not explained properly. Any suggestion?

Reply:
@rberrelleza ?? It should say `restart policy`, not `reset policy`